test/Parse/try.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -swift-version 5
```

Comment:
Can you split this into two tests, one for ambient-version mode (with no changes to the existing test) and one for `-swift-version 5`?

We still want to test the older behavior as long as it is supported.

Reply:
Yeah, I can do that.